sdk/identity/identity/src/msal/utils.ts

```diff
@@ -110,6 +111,31 @@ export const defaultLoggerCallback: (
     }
   };
 
+/**
+ * @internal
+ */
+export function convertToMSALLogLevel(logLevel: AzureLogLevel| undefined): msalCommon.LogLevel {
+ if(logLevel)
```

Comment:
I think you should be able to remove this nullcheck, the default case of the switch should take care of the undefined logLevel

Reply:
Could we call this function `getMSALLogLevel` and have it call `getLogLevel`? That way you could have

```ts
logLevel: getMSALLogLevel()
```

Since this is internal this is just a small comment that I wouldn't block on, but it is up to you